conans/client/conf/config_installer.py

```diff
@@ -247,3 +293,37 @@ def configuration_install(app, uri, verify_ssl, config_type=None,
         else:
             configs = [(c if c != config else config) for c in configs]
         _save_configs(configs_file, configs)
+    try:
+        if _get_config_install_interval(app):
+            _generate_sched_file(cache)
+    except ConanException:
+        pass
+
+
+def is_config_install_scheduled(api):
```

Comment:
All this logic seems a bit more complicated than necessary. Maybe try relying on the modified time with the following?:

- Let the sched file be empty, and use the file modified time
- Get the last modified time of the ``sched`` file with os.path.getmtime("sched")
- Do a ``os.utime("sched")`` to touch it and modified the time. (touch can create the file too)